frontend/package.json

```diff
@@ -58,6 +58,7 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/react": "^10.0.1",
```

Comment:
Let's remove `@types/*` from `dependencies`. Enzyme is an alternative of `testing-library`. Let's remove all enzyme related package from `dependencies` also.